cylc/flow/scripts/validate_install_play.py

```diff
@@ -109,5 +110,6 @@ def main(parser: COP, options: 'Values', workflow_id: Optional[str] = None):
         source=orig_source,
     )
 
+    toggle_timestamps(LOG, True)
```

Comment:
Still need to respect CLI option
```suggestion
    toggle_timestamps(LOG, options.log_timestamp)
```